manifests/prerequisites.pp

```diff
@@ -31,7 +31,7 @@
             }
         }
         'Debian' : {
-            if ($one::params::one_repo_enable == 'true') {
+            if ($one::params::one_repo_enable == true) {
```

Comment:
Ahh that's it - when I saw the lint tests throw dozens of WARNINGs and a couple of ERRORs, I ran `puppet-lint -f` over all the `.pp` manifests. It looks like `puppet-lint` was a little over-zealous in its optimisation. Thanks for the explainer.